cmake/EICreconConfig.cmake.in

```diff
@@ -13,20 +13,23 @@ find_package(EDM4HEP REQUIRED)
 find_package(EDM4EIC REQUIRED)
 find_package(DD4hep REQUIRED)
 find_package(podio REQUIRED)
+find_package(IRT REQUIRED)
 
 set(EDM4EIC_INCLUDE_DIR ${EDM4EIC_DIR}/../../include)
+set(IRT_INCLUDE_DIR ${IRT_DIR}/../../include)
```

Comment:
In principle, this would usually be defined in https://github.com/eic/irt/blob/main/cmake/IRTConfig.cmake.in, but I don't have strong objection to this way.

Reply:
Good point, I should do it that way instead (and probably so should EDM4eic).